.github/workflows/scan_images.yaml

```diff
@@ -0,0 +1,64 @@
+name: Scan Images In Bahmni Namespace
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  scan-images:
+    name: Scan Images In Bahmni Namespace
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Get current date
+        id: date
+        run: |
+          date=$(date +'%d-%m-%Y')
+          echo "date=$date" >> $GITHUB_OUTPUT
+      - name: Get Scanner Script and Template File
+        run: |
+          curl -o image-scanner.sh https://raw.githubusercontent.com/Bahmni/bahmni-infra-utils/main/image-scanner.sh
+          chmod +x image-scanner.sh
+          curl -o html.tpl https://raw.githubusercontent.com/Bahmni/bahmni-infra-utils/main/html.tpl
+      - name: Scan Images In Bahmni Namespace
+        run: |
+          ./image-scanner.sh
+      - name: Upload Report Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: image-scan-summary
+          path: container-scanner-reports/bahmni-${{ steps.date.outputs.date }}
+
+  save-scan-summary:
+    name: Save Scan Summary
+    needs: scan-images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout security-private repo
+        uses: actions/checkout@v4
+        with:
+          repository: Bahmni/security-reports
+          token: ${{ secrets.BAHMNI_PAT }}
+      - name: Get current date
+        id: date
+        run: |
+          date=$(date +'%d-%m-%Y')
+          echo "date=$date" >> $GITHUB_OUTPUT
+      - name: Download Cluster Summary
+        uses: actions/download-artifact@v4
+        with:
+          name: image-scan-summary
+          path: image-scan-report/${{ steps.date.outputs.date }}
+      - name: Commit and push changes
```

Comment:
Same here, we can write this as a step in the scan job itself, this will reduce a lot of redundant steps